class-wc-siftscience-dependencies.php

```diff
@@ -98,8 +98,17 @@ private function build( $class ) {
 			}
 
 			$args = array();
-			foreach ( $c->getParameters() as $p ) {
-				$args[] = $this->get( $p->getType()->getName() );
+
+			$php_version = explode( '.', phpversion() );
+
+			if ( 7 >= $php_version[0] && 1 <= $php_version[1] ) {
```

Comment:
I THINK `7 >= $php_version[0]` should be `7 <= $php_version[0]`, right?

Reply:
Think it loudly if major version less than 7 it should run the new code

Reply:
Actually, I think we're both wrong :-) (your logic is not right, but my solution is also not right)

For example: 
- 7.0 -> true and false --> false (this is correct)
- 7.1 ->  true and true --> true (this is correct)
- 8.0 -> false and true --> false (this is not correct)
- 5.0 -> true and false --> false (this is correct)
- 5.1 -> true and true --> true (this is not correct)

Reply:
I think the solution would need to be something like: 

`(7 < $php_version[0]) || (7 === $php_version[0] && 1 <= php_version[1])`

But an easier to read solution in my opinion is:

```
$val = (100 & $php_version[0]) + $php_version[1];
if (71 <= $val) {.....
```